### ISSUE TITLE
About the performance of the model

### ISSUE THREAD

**Issue by lpf121924** [LMissher/ST-GRAT#2]
I have used your model on some data sets, and the performance is a bit strange, can you help provide your results?

**Comment by LMissher** [comment]
My results on the PEMS04 and PEMS08 data sets are fairly normal, but the results on METR-LA and PEMS-BAY are not so good. I am trying to solve this problem.

**Comment by lpf121924** [comment]
In line 188：    ' loss = _compute_loss(y*std+mean, y_hat*std+mean) '
ON the data sets pems-bay and metr-la，should it be changed to :  'loss = _compute_loss(y , y_hat * std + mean) '

**Comment by LMissher** [comment]
I think line 188 should not be modified, because the groud truth in utils.py is also normalized, you can see lines 118-120 in utils.py.

**Comment by lpf121924** [comment]
Thank you for your reply

**Comment by CYBruce** [comment]
@LMissher How to reproduce the same result as the paper on META-LA  dataset? I ran this code but it performs really bad.

**Comment by CYBruce** [comment]
Is this the official repo of the ST-GRAT paper?

**Comment by zouguojian** [comment]
can you share your final weighs? there are some of problems for your model.

**Comment by zouguojian** [comment]
my results on META-LA is mae is : 4.676684
rmse is : 12.283457
mape is : 0.101710
r is : 0.850775
r$^2$ is : 0.709624

**Comment by littlebird-maker** [comment]
excuse me, could you please explain some code in your model.py/SpatioAttention ?  i think the code of compute the eso and es is not the same as the paper said

**Comment by littlebird-maker** [comment]
and what the shape of dataset to produce SE is ?
